app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt

```diff
@@ -434,9 +418,7 @@ class TopicLessonsFragmentTest {
         )
       ).perform(click())
       onView(withId(R.id.story_summary_recycler_view)).perform(
-        scrollToPosition<RecyclerView.ViewHolder>(
-          2
-        )
+        actionOnItemAtPosition<RecyclerView.ViewHolder>(2, scrollTo())
```

Comment:
Why not use ``scrollToPosition`` instead? Is this what you meant in our meeting today that ``scrollToPosition`` isn't working? What is it that you're observing?

Reply:
Yes. 
While using `scrollToPosition `, the recyclerview item is not `CompletlyDisplayed` after scrolling, and because of which robolectric is not able to perform and ViewAction. But using ViewActions.scrollTo, it seems working and `CompletlyDisplayed` too.

Reply:
Strange. What's the difference between the two?